src/components/LeftNavBar.js

```diff
@@ -1,34 +1,38 @@
 import { BsFillPersonFill, TbTextDirectionRtl, TbLayoutDashboard,
   IoNotificationsSharp, MdBackupTable, GiBulletBill, SiVirtualbox, RxDashboard, ImEnter, AiOutlineForm,
 } from '../utils/icons';
+import "./LeftNavBar.css"
 
 export default function LeftNavBar() {
- let navItemCSS = 'flex items-center py-3 hover:bg-neutral-900 rounded-lg active:bg-pink-600'
-
+ let navItemCSS = 'flex items-center py-3 hover:bg-neutral-900 rounded-lg active:bg-pink-600 mx-4'
+ let iconCSS = "p-1 text-3xl"
+ 
   return (
-    <div className="w-screen grid grid-cols-2 border-[2px] border-white ">
-      <div className="w-[180px] h-[calc(100%-4px)] fixed top-[2px] left-5 border-[2px] border-black bg-gradient-to-b from-neutral-800 to-neutral-950 rounded-lg">
-        <div className="mx-4 text-white text-sm">
-          <div className='flex items-center py-3 border-b-2 mb-2'><TbLayoutDashboard className="p-1" />Material Dashboard </div>
-          <div className="block h-[calc(50vh)] overflow-auto overscroll-y-auto">
+    <div className="w-screen grid grid-cols-2 border-[2px] border-white">
+      <div className="w-[250px] h-[calc(100%-40px)] fixed top-[2px] left-5 border-[2px] border-black bg-gradient-to-b from-neutral-800 to-neutral-950 rounded-lg align-tems">
+        <div className=' text-white border-yellow-400 border-b py-10 flex justify-center items-center mb-2'>
+           <div className='flex items-center mb-2 mx-4 absolute z-40'><TbLayoutDashboard className={iconCSS} />Material Dashboard 2</div>
+        </div>
+        <div className=" text-white text-sm">
+          <div class="contentNav">
```

Comment:
This one would naturally adjust to what i had in my previous comment (my suggest code was done without actually testing if i'm right, so of course, you should vet the code yourself!)

Reply:
Added my own code after transferring the CSS to the config file.